pkg/collector/corechecks/servicediscovery/events.go

```diff
@@ -40,6 +40,7 @@ type eventPayload struct {
 	PID                 int      `json:"pid"`
 	CommandLine         []string `json:"command_line"`
 	RSSMemory           uint64   `json:"rss_memory"`
+	CPUUsage            float64  `json:"cpu_cores"`
```

Comment:
Why not use the same name for the Go field as the JSON field (even if it's an odd name) like all the other fields, to avoid confusion?